user/usertests.c

```diff
@@ -1977,7 +1977,7 @@ forktest(char *s)
 
   if(n == N){
     printf("%s: fork claimed to work 1000 times!\n", s);
-    exit(1);
```

Comment:
Why not exit(0)?

Reply:
Because it should not end just yet we still need to wait for all the processes to finish